src/Common/Auth/OpenIDConnect/Repositories/AccessTokenRepository.php

```diff
@@ -21,6 +21,25 @@
 class AccessTokenRepository implements AccessTokenRepositoryInterface
 {
 
+    /**
+     * Returns the token expiration date for the given token id.  If $userId is null it will return the token for a
+     * confidential_client access token
+     * @param $tokenId string The access token id
+     * @param $clientId string The client id
+     * @param number|null $userId  The id of the openemr user the token corresponds to
+     * @return string|null The expiration date or null if there was no token found
+     */
+    public function getTokenExpiration($tokenId, $clientId, $userId = null)
+    {
+        if (empty($userId)) {
```

Comment:
Maybe i am misreading the code, but won't all tokens have a user id now that system is binded to the system user?

Reply:
Yea, good catch.  This was left over from when I didn't have the api system user.

Reply:
Fixed this.